client-src/elements/chromedash-admin-blink-component-listing.ts

```diff
@@ -81,49 +92,51 @@ export class ChromedashAdminBlinkComponentListing extends LitElement {
       `,
     ];
   }
-  /** @type {import('chromestatus-openapi').DefaultApiInterface} */
-  _client;
-
-  static get properties() {
-    return {
-      editing: {type: Boolean, reflect: true},
-      component: {type: Object},
-      index: {type: Number},
-      usersMap: {type: Object},
-      id: {type: Number},
-      name: {type: String},
-      subscriberIds: {type: Array},
-      ownerIds: {type: Array},
-    };
-  }
 
-  constructor() {
-    super();
-    this._client = window.csOpenApiClient;
+  @property({type: Boolean, reflect: true})
+  editing = false;
+  @property({type: Object})
+  component = {};
+  @property({type: Number})
+  index = 0;
+  @property({type: Object})
+  usersMap: Map<number, User> = new Map();
+  @property({type: Number})
+  componentId = 0;
```

Comment:
Why did you rename `id` to `componentId`? It's not necessarily wrong, but surprising as part of the Typescript refactoring.

Reply:
it was because there's user.id down below, and TypeScript does not allow this overlap

Reply:
oh sorry i messed this up with another error, I changed this because `Property 'id' in type 'ChromedashAdminBlinkComponentListing' is not assignable to the same property in base type 'LitElement'.` as it extends from litelement

Reply:
Aha! Because LitElement is an HTMLElement, and `id` there is https://developer.mozilla.org/en-US/docs/Web/API/Element/id. Yep, good change.